t2.cpp

```diff
@@ -0,0 +1,6 @@
+sdjfsjsdjfsd;
+fsdfsdfsf;
+sdfdsfdsf;
+sdfsdfsdf;
+dsfsdfs;
```

Comment:
variable 'dsfsdfs' is non-const and globally accessible, consider making it const [cppcoreguidelines-avoid-non-const-global-variables]